docker-compose.e2e.yaml

```diff
@@ -38,6 +38,7 @@ services:
       - shared_contracts_e2e:/build:rw
       - ./hardhat/contracts:/hardhat/contracts:ro
       - ./hardhat/scripts:/hardhat/scripts:ro
+      - ./hardhat/tasks:/hardhat/tasks:ro
```

Comment:
I don't have such `file/directory`, does it belong to this PR?

Reply:
I just get it from the develop branch, but I can remove it since will be add by a merge with develop.

Reply:
Yea this is from another branch that is already merged into `develop`